src/demo/index.html

```diff
@@ -408,9 +420,33 @@ <h1>Symphony Electron API Demo</h1>
             window.postMessage({ method, data }, '*');
         };
 
+        const getRTCLogs = () => {
+            if (document.getElementById('rtc-log-filename2').value == '') {
+                const filename = document.getElementById('rtc-log-filename').value;
+                const contents = document.getElementById('rtc-log-contents').value;
+                return [ { filename, contents }, ];
+            } else {
+                const filename1 = document.getElementById('rtc-log-filename').value;
+                const contents1 = document.getElementById('rtc-log-contents').value;
+                const filename2 = document.getElementById('rtc-log-filename2').value;
+                const contents2 = document.getElementById('rtc-log-contents2').value;
+                return [ { filename: filename1, contents: contents1 }, 
+                    { filename: filename2, contents: contents2 }, ];
+            }
+        }
+
         window.addEventListener('message', (event) => {
             const { data, method } = event.data;
             switch (method) {
+                case 'collect-logs':
+                    if( data === 'RTC' ) {
```

Comment:
Are you broadcasting one message for each registered provider? Wouldn't it be easier if each provider responds to the same message? It's a broadcast, so it will reach all.

I don't think there is a case where you wouldn't want everyone to provide logs, you would always send to everyone anyway.
The implementation is simpler, because you don't have to check that the message is for you.

Reply:
Not really sure what this means, I was pretty much just following existing patterns. What change do you propose?

Reply:
Now you pass `postMessage('collect-logs') once for each listener, and all listeners will receive that message. Each listener is expected to only reply to one of them.

I propose that you call `postMessage('collect-logs')` only once. All listeners will receive that message, and they can respond with `send-logs`.